Client/Frontend/Browser/TabsBar/TabsBarViewController.swift

```diff
@@ -60,11 +60,13 @@ class TabsBarViewController: UIViewController {
     }
   }
 
-  fileprivate weak var tabManager: TabManager?
-  fileprivate var tabList = WeakList<Tab>()
+  private weak var tabManager: TabManager?
+  private weak var browserViewController: BrowserViewController?
+  private var tabList = WeakList<Tab>()
 
-  init(tabManager: TabManager) {
+  init(tabManager: TabManager, browserViewController: BrowserViewController) {
```

Comment:
Perhaps it would be better to add new delegate method to `TabsBarViewControllerDelegate`?
BVC implements it and this way we can avoid passing entire bvc to this class

Reply:
Yeah I thought that, I was thinking making bvc bigger or passing the reference. But will do the change.

Reply:
Fix at https://github.com/brave/brave-ios/pull/5232/commits/1f4d62bd0ad0ce2a442cf89302d883c134cfc332